src/components/display/fewlines/Header/Header.tsx

```diff
@@ -8,7 +9,11 @@ export const Header: React.FC = () => {
 
   return (
     <Flex>
-      <img width="90" src={theme.logo} aria-label="Fewlines logo" />
+      <Link href="/account">
+        <a>
+          <img width="90" src={theme.logo} aria-label="Fewlines logo" />
```

Comment:
Can we adapt the `arial-label`? 😇

Reply:
Is `"Provider logo"` good enough, or do you want me to use the theme value? (i.e. fewlines | decatlon).

Reply:
Let's go with `Logo` for now and we'll make it better another time 😁